src/main/webapp/js/models/question.js

```diff
@@ -32,7 +32,7 @@ var QUESTION_TYPE_PHRASE = 3;
 var QUESTION_TYPE_SEQUENCE = 4;
 var QUESTION_TYPE_SET = 5;
 
-var SEQUENCE_0 = "0";
+// var SEQUENCE_0 = "0";
```

Comment:
I assume this is now defined elsewhere.. you can delete this commented out code..